resources/build/ci/trigger-build-bot.inc.sh

```diff
@@ -21,19 +21,20 @@
 function test_bot_check_pr_body() {
   local PRNUM=$1
   local prinfo="$2"
+  local prbody prTestCommand prTestBody
 
   set -o noglob
   IFS=$'\n'
-  local prbody="$(echo "$prinfo" | "${JQ}" -r '.body')"
-  local prTestCommand="$(echo "$prbody" | grep 'Test-bot:' | cut -d: -f 2 - | cut -d' ' -f 1 -)"
-  local prTestBody="$(echo "$prbody" | grep -i '# User Testing')"
+  prbody="$(echo "${prinfo}" | "${JQ}" -r '.body')"
+  prTestCommand="$(echo "${prbody}" | grep 'Test-bot:' | cut -d: -f 2 - | cut -d' ' -f 1 -)"
+  prTestBody="$(echo "${prbody}" | grep -i '# User Testing')"
   unset IFS
   set +o noglob
 
-  if ([[ "$prTestCommand" == skip ]] || [[ -z "${prTestCommand// }" ]]) && [[ -z "${prTestBody// }" ]]; then
+  if { [[ "${prTestCommand}" == skip ]] || [[ -z "${prTestCommand// }" ]]; } && [[ -z "${prTestBody// }" ]]; then
```

Comment:
I'm unclear how the curly brace grouping works in an if statement

Reply:
That was a suggestion from shellcheck ([SC2235](https://www.shellcheck.net/wiki/SC2235)). Serves the same purpose as the parens, but doesn't create a subshell and so is faster.